tests/framework/framework.go

```diff
@@ -76,34 +80,42 @@ func init() {
 // NewFramework makes a new framework and sets up the global BeforeEach/AfterEach's.
 // Test run-time flags are parsed and added to the Framework struct.
 func NewFramework(prefix string) *Framework {
+	var err error
```

Comment:
I think it's probably better to add an err return to this func. Then we don't have to think about calling Fail or Expect -- we just return the error to the caller.

Reply:
Sounds good, will change.